ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -890,6 +909,10 @@ generator genBlock m@Model {..} = At <$> frequency
     , (if empty then 1 else 10, return GetMaxSlotNo)
     , (if empty then 1 else 10, genGetIsValid)
 
+    , (case loe of
+         LoEDisabled   -> (0, return $ UpdateLoE $ AF.Empty AF.AnchorGenesis)
```

Comment:
Wouldn't setting the frequency to 0 be equivalent to not generating this case at all?

Reply:
Yes, we just use `0` here for convenience as we are inside of a list literal here; a similar pattern is used in a few other places; I slightly refactored this to at least not use a bogus value on the right.
```diff
diff --git a/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs b/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs
index 510ebac6df..184f018a4b 100644
--- a/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs
+++ b/ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs
@@ -909,9 +909,10 @@ generator loe genBlock m@Model {..} = At <$> frequency
     , (if empty then 1 else 10, return GetMaxSlotNo)
     , (if empty then 1 else 10, genGetIsValid)
 
-    , (case loe of
-         LoEDisabled   -> (0, return $ UpdateLoE $ AF.Empty AF.AnchorGenesis)
-         LoEEnabled () -> (if empty then 1 else 10, UpdateLoE <$> genLoEFragment))
+    , let freq = case loe of
+            LoEDisabled   -> 0
+            LoEEnabled () -> if empty then 1 else 10
+      in (freq, UpdateLoE <$> genLoEFragment)
 
     -- Iterators
     , (if empty then 1 else 10, uncurry Stream <$> genBounds)
```